### PR TITLE
Use proper Unicode box-drawing characters for state machine diagram

### PR DESCRIPTION
**Pull request by das-g**
Use proper [Unicode box-drawing characters](https://en.wikipedia.org/wiki/Box-drawing_character#Unicode) for state machine diagram.

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -11,21 +11,21 @@ The clients send commands to the server and receive the current gamestate.
...
-                 | |                                              | |
-                 | | +----------------+   +---------------------+ | |
-                 | +->Command(Command)+--->UpdateGamestate(Game)+-+ |
-                 |   +----------------+   +---------------------+   |
-                 |                                                  |
-                 |   +-------------------------+                    |
-                 +--->ClientReconnect(ClientId)+--------------------+
-                     +-------------------------+
+    ┏━━━━━━━┓   ┏━━━━━━━━━━━┓   ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+    ┃*start*┠───►ClientHello┠───►ServerHello(ClientId, World)┠─╮
```
Comment (rnestler) [inline docs/design.md]: Can we make the arrows so that they don't enter the box but are adjacent?

**Comment by rnestler** [comment]
Nice! Looks way better IMO :sparkles:

File: docs/design.md (hunk 1 above)
Comment (das-g) [reply docs/design.md]: sure: 55eececd51ae78141e5e29db3a7fe0daefaeff6a

**Comment by rnestler** [comment]
One minor nitpick: Could you start the commit messages with an uppercase letter? I try to follow the recommendations from https://chris.beams.io/posts/git-commit/. I could also change it for you if you don't mind :wink:

**Comment by rnestler** [comment]
>One minor nitpick: Could you start the commit messages with an uppercase letter? I try to follow the recommendations from https://chris.beams.io/posts/git-commit/. I could also change it for you if you don't mind :wink:

Looking through the commit history I noticed that I didn't always follow the recommendations myself :scream:

**Comment by rnestler** [comment]
Thanks again!

**Pull request merged**